tests/integration-tests/pom.xml

```diff
@@ -370,12 +370,6 @@
          <artifactId>woodstox-core-asl</artifactId>
          <version>4.4.0</version>
          <scope>test</scope>
-         <exclusions>
```

Comment:
Why did you remove that exclusion ?

Reply:
I have reverted it. I was missing that dependency in the dependency tree